std/socket.d

```diff
@@ -63,7 +62,8 @@ version(Windows)
     pragma (lib, "ws2_32.lib");
     pragma (lib, "wsock32.lib");
 
-    private import core.sys.windows.windows, core.sys.windows.winsock2, std.windows.syserror;
+    public import core.sys.windows.winsock2;
+    private import core.sys.windows.windows, std.windows.syserror;
```

Comment:
`INADDR_LOOPBACK` is publicly used in `std.net.curl` module. To make it visible through `std.socket` module, publicly import `core.sys.windows.winsock2` here.